chart/README.md

```diff
@@ -540,6 +540,29 @@ Some example commands are shown below.
    .
    ```
 
+#### Installing from remote repository
+```
+helm repo add sonatype https://sonatype.github.io/helm3-charts/
+
+helm install --namespace staging mycluster --dependency-update \
+--set serviceAccount.create=true \
+--set iq_server.serviceAccountName=nxlc-default-l8qiag0c-service \
+--set serviceAccount.annotations."eks\.amazonaws\.com/role-arn"="arn:aws:iam::552183322382:role/nxlc-default-l8qiag0c-role" \
```

Comment:
I wonder if we're not exposing sensitive data in this and other examples.

Reply:
> I wonder if we're not exposing sensitive data in this and other examples.

I think we are. I don't think we need all these annotation values or aws specific stuff. Something minimal should be good enough (as in if we are installing in minikube)

Reply:
I removed extra parameters.